cypress.json

```diff
@@ -2,7 +2,7 @@
   "baseUrl": "https://qa.nrs-ghtr.org.uk/",
   "accessibility": false,
   "reporter": "spec",
-  "reporterEnabled": "spec",
+  "reporterEnabled": "junit,spec",
   "reporterOptions": {
     "mochaFile": "results/test-output-[hash].xml",
```

Comment:
Oh so you need both.. the command line switch and this entry in the config file?
I tried the `--reporter` switch on the command line alone and didn't got anything.

Reply:
I've ran this branch and test results are not being published still. 

The step "CmdLine" in this job runs a `find .` in the working directory, I can't seem to find any occurrences of `test-output-*.xml` in the file list.